### PR TITLE
Issue#20-Tracking-Page

### LINKED ISSUE #20: [Order Process] Order Tracking Panel
## User Story
As a **User**, I would like **to be able to track what state my order is in and where it is along with an estimated delivery time** so that **I can anticipate when my order will arrive**.

## Detailed Description
Page that gets navigated to after payment submission where you can see what stage your order is in as well as a summary of your order.

## Acceptance Criteria
- [x] Create Order table in db
- [x] Submitting payment from checkout screen pushes order to db
- [x] order progress is tracked and displayed until marked as delivered by chef

### PR DESCRIPTION
**Pull request by jcgubbi**
Fixes #20 

Adds basic tracking screen that progresses every 0.5 seconds. Used icons from https://fontawesome.com/icons?d=gallery&p=2&m=free for images. Have subtotal at bottom of screen and a button to go back to home page. 

Simulator Screen Shot - iPhone 12 Pro - 2021-02-27 at 13 19 26

### CODE REVIEW

**Review by brendon-ng — APPROVED** [review]
LGTM

**Comment by brendon-ng** [comment]
possible extension of this story is to let users go back into the app and browse/use the app normally while the order is being fulfilled. 
And have a button in Cart to view the tracking order panel if an order is currently unfulfilled

File: app/screens/Tracking.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            <CheckoutSubtotal></CheckoutSubtotal>
+        </View>
+        <View style={styles.paymentTab}>
+            <Button
+              type="solid"
+              title="Done"
+              titleStyle={styles.doneText}
+              buttonStyle={styles.doneButton}
+              onPress={() => {
+                // Do a thing
```
Comment (shikham-8) [inline app/screens/Tracking.js:80]: For the onPress of the "Done" button, can you also clear the shopping cart? 
```global.cart = []``` (or add a method into the ShoppingCart to remove all previous entries)

Comment (brendon-ng) [reply app/screens/Tracking.js:80]: I will handle this in review prompt issue #22

**Pull request merged**